lib/serializables/v1/ssh.js

```diff
@@ -62,6 +62,9 @@ function SshFactory (Promise, Serializable, encryption) {
         if (this.password) {
             this.password = 'REDACTED';
         }
+        if (this.publicKey) {
+            this.publicKey = 'REDACTED';
+        }
```

Comment:
Public keys do not need to be encrypted or redacted.  These can be safely revealed to anyone.